.circleci/config.yml

```diff
@@ -54,6 +54,27 @@ jobs:
             cargo test --lib --all-features
       # No need to save this in cache
 
+  tests_ffi:
+    docker:
+      - image: zondax/rust-ci:latest
+    steps:
+      - checkout
+      - restore-sccache-cache
+      - run:
+          name: Initial configuration
+          command: |
+            sudo apt-get install -y default-jdk g++ gcc valgrind snapd
+            wget -O sdk.install.sh "https://get.sdkman.io"
+            sudo bash sdk.install.sh
+            sudo source "/root/.sdkman/bin/sdkman-init.sh"
+            sudo sdk install kotlin
+            rustup install nightly
+            cargo +nightly install cbindgen
```

Comment:
These dependencies could be provided by a custom development image

Reply:
yes, I will open an issue for this https://github.com/Zondax/docker-builder/issues/2
we can prioritize that later